fe/fe-core/src/main/java/org/apache/doris/analysis/ShowColumnStmt.java

```diff
@@ -98,8 +106,14 @@ public void analyze(Analyzer analyzer) throws AnalysisException {
             tableName.setDb(db);
         }
         tableName.analyze(analyzer);
-        // disallow external catalog
-        Util.prohibitExternalCatalog(tableName.getCtl(), this.getClass().getSimpleName());
+
+        if (!Env.getCurrentEnv().getAccessManager()
```

Comment:
Better to use this method `public boolean checkTblPriv(UserIdentity currentUser, String ctl, String db, String tbl, PrivPredicate wanted)` to and pass all `catalog`, `database` and `table` info

Reply:
fixed